pkg/discovery/worker/garbage_collector.go

```diff
@@ -110,3 +156,78 @@ func (g *GarbageCollector) isLeakedPod(pod api.Pod) bool {
 
 	return false
 }
+
+// The controllers cleanup is supposed to happen only at the startup
+// We fail in case of errors to force kubeturbo to restart and try the cleanup again
+func (g *GarbageCollector) CleanupControllers() error {
+	if err := g.cleanupParents(); err != nil {
```

Comment:
Suggest renaming to something like `revertScheduler`

Reply:
Done!